File: server/ncurses/button.py
from ncurses.widget import Widget

class Button(Widget):

    def __init__(self, text, x, y):
        self.x = x
        self.y = y
        self.text = text

    def draw(self, screen):
        screen.move(self.x, self.y)
        screen.printw("[" + self.text + "]")

File: server/bak/ncurses/button.php
<?php

require_once('ncurses/widget.php');

class Button extends Widget {
    private $text;

    function __construct($text, $x, $y) {
        $this->text = $text;
        $this->x = $x;
        $this->y = $y;
    }

    function __destruct() {
    }

    function draw($screen) {
        $screen->move($this->x, $this->y);
        $screen->printw("[" . $this->text . "]");
    }
}

?>